http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GceInterpreter.scala

```diff
@@ -370,6 +370,9 @@ class GceInterpreter[F[_]](
         metadataToAdd = metadata,
         metadataToRemove = Set("startup-script-url")
       )
+      // This sleep is added here because as of 2/1/2022, we started seeing increased amount of `RESOURCE_NOT_READY` error.
+      // Google support says this is due to the setMetadata API and start VM API is called too close to each other.
```

Comment:
It may be good to reference the Jira ticket and/or the Google ticket for posterity.